### ISSUE TITLE
Redesign/Style the new contest page

### ISSUE THREAD

**Issue by Signior-X** [KamandPrompt/CodeManiacs#153]
Currently the creating new contest page does not seem much user friendly and good. 
![image](https://user-images.githubusercontent.com/56730716/101639243-0d9dab00-3a55-11eb-85c2-f38217327266.png)

Make it look better using css so that it will be more appealing to see.
Suggestions are always welcome!

**Comment by pranshukharkwal** [comment]
Also do the same design changes in the edit contest page : /admin/edit-contest/round1

**Comment by Ayushgup85** [comment]
Can i work on this issue?

**Comment by Shagufta08** [comment]
I would like to work on this issue?

**Comment by pranshukharkwal** [comment]
@Ayushgup85 You can work on this issue. 
@Shagufta08 Please pick some other issue.

**Comment by danish1804** [comment]
Plss can you assign this issue to me @Shagufta08 @Ayushgup85 @sahilarora535 i can work on the UI part

**Comment by Signior-X** [comment]
@Ayushgup85 are you working on this?

**Comment by Ayushgup85** [comment]
Yes i am working on this issue .I will submit it asap.Actually my exams are going on.

**Comment by NikhilSharmaWe** [comment]
looking forward to contribute on this issue

**Comment by NikhilSharmaWe** [comment]
i am unable to find the html of this page can anyone help me how to find it

**Comment by vsvipul** [comment]
@kpbot claim
